SessionUIKit/Components/ConfirmationModal.swift

```diff
@@ -275,9 +275,11 @@ public class ConfirmationModal: Modal, UITextFieldDelegate {
         
         titleLabel.isAccessibilityElement = true
         titleLabel.accessibilityIdentifier = "Modal heading" // stringlint:disable
+        titleLabel.accessibilityLabel = titleLabel.text
         
         explanationLabel.isAccessibilityElement = true
         explanationLabel.accessibilityIdentifier = "Modal description" // stringlint:disable
+        explanationLabel.accessibilityLabel = explanationLabel.text
```

Comment:
@RyanRory Just wondering with these - wouldn't the screen reader already read out the content of the labels without this? (Or is this change specifically for the automated UI tests?)

Reply:
This is a change for automated test. As far as I know the screen reader should be able to read out the text, and the `accessibilityLabel` should be set to the text as default, but it seems like it doesn't and Appium can't get the `accessibilityLabel` correctly, so I have to set them explicitly.

Reply:
Yea no worries - bit of a shame (as ideally we wouldn't custom set this and let the OS handle it) but I guess at the moment we aren't really screen reader accessible so when we eventually take a proper look at that we can change this if it causes issues